api/v1alpha2/cloud/types.go

```diff
@@ -51,9 +51,22 @@ type Config struct {
 // ProviderConfig defines any extra information used to configure
 // the vSphere external cloud provider
 type ProviderConfig struct {
+	CCM CCMConfig `json:"ccmConfig,omitempty"`
+	CNS CNSConfig `json:"cnsConfig,omitempty"`
+}
+
+type CCMConfig struct {
 	Image string `json:"image,omitempty"`
```

Comment:
Hi @andrewsykim,

Please consider renaming `Image` to `ControllerImage` to match the `CNSConfig` or renaming `CNSConfig.ControllerImage` to `CNSConfig.Image`.

I personally think `CCMConfig.Image` should be `CCMConfig.ControllerImage` for consistency.

Reply:
yeah I like that too, will update